src/extra/helpers/highlight.cpp

```diff
@@ -0,0 +1,59 @@
+#include "helpers.hpp"
+
+namespace shards {
+namespace Helpers {
+
+using namespace gfx;
+
+struct HighlightShard : public BaseConsumer {
+  // TODO: Merge with DrawShard type
+  static inline shards::Types SingleDrawableTypes = shards::Types{gfx::Types::Drawable, gfx::Types::DrawableHierarchy};
+  static inline Type DrawableSeqType = Type::SeqOf(SingleDrawableTypes);
+  static inline shards::Types DrawableTypes{gfx::Types::Drawable, gfx::Types::DrawableHierarchy, DrawableSeqType};
+
+  static SHTypesInfo inputTypes() { return DrawableTypes; }
+  static SHTypesInfo outputTypes() { return CoreInfo::NoneType; }
+  static SHOptionalString help() { return SHCCSTR("Queues a draw operation to highlight a drawable"); }
+
+  SHTypeInfo compose(const SHInstanceData &data) {
+    if (data.inputType.basicType == SHType::Seq) {
+      OVERRIDE_ACTIVATE(data, activateSeq);
+    } else {
+      OVERRIDE_ACTIVATE(data, activateSingle);
+    }
+    return CoreInfo::NoneType;
+  }
+
+  SHVar activateSingle(SHContext *shContext, const SHVar &input) {
+    Context &helperContext = getContext();
+
+    SHTypeInfo inputType{.basicType = SHType::Object,
+                         .object = {.vendorId = input.payload.objectVendorId, .typeId = input.payload.objectTypeId}};
```

Comment:
Could probably use the new helper Type::Object.

```suggestion
    auto inputType = Type::Object(input.payload.objectVendorId, input.payload.objectTypeId);
```